### ISSUE TITLE
Trigger question

### ISSUE THREAD

**Issue by scottlep** [Shufflepuck/SplashBuddy#54]
This is a usage question.

We are a Managed Service Provider that often implements Jamf into new clients with existing computers. We would be enrolling all existing computers in to Jamf and also implementing DEP and Zero Touch deployment. I set up SplashBuddy for some testing. I don't want to use the trigger "Enrollment Complete" because I only want the policies installed on computers that have been wiped and the OS reinstalled. I don't want apps reinstalling on the existing computers. What trigger could I use to scope machines that have been wiped and re-enrolled?

**Comment by killahquam** [comment]
Build a dummy package and deploy it to you existing fleet then scope splash
buddy policies to all things where that package is missing.
Keep the trigger on enrollment and frequency on ongoing. In my setup the
apps are scoped for computers missing those apps so the chances of that
happening are slim. If you just concerned about splashbuddy just do
touch /Users/Shared/.SplashBuddyDone and splashbuddy won't launch if
installed.

**Comment by hgriffith-ewu** [comment]
You could also set up a Smart Computer Group with a criteria set to Last Enrollment date after all of their existing computers are enrolled.  Any new computers or old computers that have been wiped and rejoined will then get the Smart Buddy profile moving forward.

**Comment by scottlep** [comment]
The problem is that we often have computers that were already in use enrolling the same time as brand new/wiped computers. So we would have no way to deploy the dummy package to the existing before the enrollment. Last Enrollment date Smart Group doesn't work either. Only thing I can think of is to look for something that exists consistently across the existing and scope to that.

**Comment by killahquam** [comment]
If you have the serials of all the existing macs, or the new ones you
could build a smart group with that too.



2017-11-18 12:17:43 +0000 scottlep <notifications@github.com> :

**Comment by scottlep** [comment]
Thanks. We don't have the serials of the existing computers since the client is new to us and they don't have great inventory records.  I might be able to do that in reverse and make a group with the serials that I know are new purchases or wiped.

As a work around for now I added the text "New Computer" as the "Purchasing Contact" in the Pre Stage Enrollment>Purchasing config. I then have a smart group that looks for that and then I used that smart group as the scope
